translation.rst

```diff
@@ -362,6 +343,41 @@ Translations in Templates
 Most of the time, translation occurs in templates. Symfony provides native
 support for both Twig and PHP templates.
 
+.. _translation-filters:
+
+Using Twig Filters
```

Comment:
Filters moved before tags as this is the recommended way IMO (because of the escaping difference).